fixtures/merkyl.py

```diff
@@ -87,3 +87,11 @@ def test_test(merkyl_inspector):
                 print(merkyl_inspector.get_log('/path/to/log/file'))
     """
     return MerkylInspector(request)
+
+
+@pytest.yield_fixture(scope="session", autouse=True)
+def merkyl_setup(request):
+    fire_art_hook(request.config, 'setup_merkyl', ip=urlparse(env['base_url']).netloc)
```

Comment:
this should use the appliance fixture, we may need to review appliance object initialization